nls/kappnav.properties

```diff
@@ -88,6 +88,12 @@ table.header.component = Component
 table.header.age = Age
 table.header.console = Console
 
+# Messages for when the resource table is empty 
+table.empty = There are no {0} here. Go add one with 
```

Comment:
You should pass in a 2nd parameter for the button name.  For example, There are no {0} here. Go add one with {1}.  Otherwise in different languages they will not be able to move {1} around.  It may not be the last part of the sentence in a different language.  For that reason you should also end the sentence with period.

Reply:
I originally wanted to do what you're suggesting(passing in a second parameter {1}), but it wouldn't work out since part of the message had to be the link that opens the 'Add Application' modal. I'm working with the designers to come up with a message that would end with the link text,  'Add [resource_name], being its own sentence.

Reply:
![image](https://user-images.githubusercontent.com/6720263/72169498-d500d780-3394-11ea-8b78-669cf675a88d.png)
This is what was agreed on.